### DISCUSSION TITLE
Hydration error when using <p> inside <div>

### DISCUSSION THREAD

**Discussion by arunppsg** [vercel/next.js#44521] [Help] [answered]
I was trying a simple example and my `index.tsx` looks like this:
```
export default function Home() {
  return (
    <div>
      <p>
        <h3>This is index</h3>
      </p>
    </div>
  )
}
```

When I run this code, I get an `Unhandled Runtime Error. Error: Hydration failed because the initial UI does not match what was rendered on the server`.

The solution which fixes the issue was replace `<p>` with `<div>`. But I would like to know why `<p>` tag does not work and how changing it into `<div>` tag makes it work.

**Comment by icyJoseph** [comment, marked as answer]
Hi,

The issue is actually nesting a heading tag, such as `h3`, within a `p` tag. Refer to this resource: https://caninclude.glitch.me/caninclude?child=h3&parent=p.

**Reply by arunppsg** [reply]
Ah, I get it, missed the point. Thanks @icyJoseph
